client/components/happiness-support/index.jsx

```diff
@@ -10,12 +13,23 @@ import sample from 'lodash/sample';
  */
 import Button from 'components/button';
 import Gravatar from 'components/gravatar';
+import { isHappychatAvailable } from 'state/happychat/selectors';
 import support from 'lib/url/support';
+import HappyChatButton from 'components/happychat/button';
+import HappyChatConnection from 'components/happychat/connection';
```

Comment:
Let's aim for consistency in capitalization of `Happychat`. Existing codebase seems to use lowercase `c` (dangit). Other than this file, the only exception appears to be `client/my-sites/checkout/checkout-thank-you/jetpack-thank-you-card.jsx`